human-dynamics-estimator/src/HumanDynamicsEstimator.cpp

```diff
@@ -34,18 +29,19 @@ double HumanDynamicsEstimator::getPeriod()
 //---------------------------------------------------------------------
 bool HumanDynamicsEstimator::configure(yarp::os::ResourceFinder &rf)
 {
+    std::string modelFilename = rf.findFile("humanModel");
```

Comment:
with `humanModel` parameter name I'd expect the name of the `urdf` file.
Are you sure you want to create ports with a filename as prefix?

Probably you meant `moduleName` or simply `name` ?